schema/sqitch.plan

```diff
@@ -229,3 +229,4 @@ shipping/views [shipping/views@2020-09-04] 2020-09-08T18:58:09Z Kairsten Fay <kf
 
 shipping/views [shipping/views@2020-09-08] 2020-09-14T21:35:26Z Kairsten Fay <kfay@fredhutch.org> # Add UW Reopening samples to reportable conditions view
 @2020-09-14 2020-09-14T21:40:13Z Kairsten Fay <kfay@fredhutch.org> # Schema as of 14 September 2020
+warehouse/site/data [warehouse/site/data@2020-09-14] 2020-09-22T01:53:23Z Chris Craft,,, <jccraft@chriscraft-lapdance> # Add UW reopening sites
```

Comment:
I'm not sure that your sqitch settings are configured properly. It's ok if you don't want to share your email properly, but maybe you can use your GitHub auto-reply email instead? Also, the language you use here to refer to your laptop name makes me feel uncomfortable. I do not want to commit this to our codebase. Could you please use a more professional personal identifier?

Reply:
I didn't know I had to configure sqitch! I manually updated the text in sqitch.plan with my work email address.

Reply:
I'm now configured.